### PR TITLE
gron 0.3.2 (new formula)

### PR DESCRIPTION
**Pull request by decors**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/gron.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    mkdir_p buildpath/"src/github.com/tomnomnom/"
+    ln_s buildpath, buildpath/"src/github.com/tomnomnom/gron"
+    ENV["GOPATH"] = buildpath
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-o", bin/"gron"
+  end
+
+  test do
+    assert_equal <<-EOS.undent, shell_output("echo '{\"foo\":1, \"bar\":2}' | #{bin}/gron")
```
Comment (MikeMcQuaid) [inline Formula/gron.rb]: Use `pipe_output` which means you don't need the `echo`.

**Comment by decors** [comment]
Thanks. I fixed it.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by tomnomnom** [comment]
@decors couldn't find another way to contact you, so here seems as good a place as any. A massive thank you from me, the main author of gron, for adding it to homebrew! What a lovely surprise :)

**Comment by decors** [comment]
@tomnomnom thank you for your message! gron is awesome.

**Pull request closed without merging**